Essentials/src/main/java/com/earth2me/essentials/User.java

```diff
@@ -49,8 +53,10 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
-import static com.earth2me.essentials.I18n.tl;
+import static com.earth2me.essentials.I18n.tlLiteral;
+import static com.earth2me.essentials.I18n.tlLocale;
 
+@SuppressWarnings("deprecation")
```

Comment:
Should suppress for specific calls, not for the whole class, since this would hide future instances of deprecation that we might actually want to address.